### PR TITLE
Fix bindings for drawAtlas

### PR DESCRIPTION
**Pull request by abarth**
In changing the binding systems, I broke the colors parameter to drawAtlas. It
was looking for an array of ints rather than an array of Colors. Now we use
CanvasImage, which has the proper converter.

Fixes #1137

### CODE REVIEW

**Comment by abarth** [comment]
@eseidelGoogle @vlidholt

File: sky/engine/core/painting/CanvasColor.h [hunk 1]
```diff
@@ -14,31 +14,31 @@
...
+  CanvasColor() : sk_color() { }
+  operator SkColor() const { return sk_color; }
 };
 
-template <>
-struct DartConverterTypes<CanvasColor> {
-  using ConverterType = CanvasColor;
-  using ValueType = SkColor;
-};
+// template <>
```
Comment (eseidelGoogle) [inline sky/engine/core/painting/CanvasColor.h]: Did you mean to delete this?

**Comment by eseidelGoogle** [comment]
lgtm

File: sky/engine/core/painting/CanvasColor.h (hunk 1 above)
Comment (abarth) [inline sky/engine/core/painting/CanvasColor.h]: Yes.

Comment (abarth) [inline sky/engine/core/painting/CanvasColor.h]: I'll actually delete it.

**Pull request merged**